xml/ha_migration.xml

```diff
@@ -136,47 +136,59 @@
   <itemizedlist>
    <listitem>
     <para>
-     Rolling upgrades are only supported from the GA of a product version to
-     the next service pack, and from one service pack to the next.
+     Rolling upgrades are only supported within the same major release (from the
+     GA of a product version to the next service pack, and from one service pack
+     to the next).
     </para>
    </listitem>
    <listitem>
     <para>
-     Offline migrations are required to upgrade from one major version to
-     the next major version (for example, from SLE&nbsp;HA&nbsp;11 to
-     SLE&nbsp;HA&nbsp;12) or from a service pack belonging to one major
-     version to the next major version (for example, from
+     Offline migrations are required to upgrade from one major release to
+     the next (for example, from SLE&nbsp;HA&nbsp;11 to
+     SLE&nbsp;HA&nbsp;12) or from a service pack within one major
+     release to the next major release (for example, from
      SLE&nbsp;HA&nbsp;11&nbsp;SP3 to SLE&nbsp;HA&nbsp;12).
```

Comment:
shouldn't the example be from 12 to 15? Otherwise it would look a bit outdated.

[...] for example, from SLE HA **12** to SLE HA **15** [...]

Same for the example with the SP.

Reply:
Good catch, done.